thunder/core/symbol.py

```diff
@@ -650,35 +652,18 @@ def has_tags(bsym: BoundSymbol, tags: set[OpTags]) -> bool:
 
 # Wrapper class that hashes and equates only the right hand side of a BoundSymbol for CSE.
 # That is to say, its symbol, args, and kwargs, but not its output.
-@dataclass
-class BoundSymbolRHS:
-    parent: BoundSymbol
-    _frozen_kwargs: FrozenDict
-
-    def __init__(self, parent: BoundSymbol) -> None:
-        self.parent = parent
-        self._frozen_kwargs = FrozenDict(parent._var_kwargs)
+# NOTE: once the unpack_trivial is fixed this can be changed to a default immmutable namedtuple.
```

Comment:
Let's fix this unpack_trivial problem first before introducing any changes to `BoundSymbolRHS`.

Reply:
I agree! I'll open a PR for that fix